### PR TITLE
enh(Listing): Use React.memo to avoid re-render on Listing hover

### PR DESCRIPTION
**Pull request by Thebarda**

### CODE REVIEW

File: src/Listing/index.tsx [hunk 1]
```diff
@@ -397,25 +397,27 @@ const Listing = ({
...
-                          }}
-                          color="primary"
-                        />
-                      </BodyTableCell>
-                    ) : null}
-
-                    {columnConfiguration.map((column) =>
-                      getColumnCell({ column, row }),
-                    )}
+                    <Row isHovered={hovered === row.id}>
```
Comment (bdauria) [inline src/Listing/index.tsx]: Can't we wrap React.memo around ListingRow instead?

File: src/Listing/index.tsx [hunk 2]
```diff
@@ -437,4 +439,16 @@ const Listing = ({
   );
 };
 
+interface RowProps {
+  children;
+  isHovered?: boolean;
```
Comment (kduret) [inline src/Listing/index.tsx]: isHovered can be undefined ?

File: src/Listing/index.tsx (hunk 1 above)
Comment (Thebarda) [reply src/Listing/index.tsx]: In this cas, we have to check when each prop changes

File: src/Listing/index.tsx (hunk 2 above)
Comment (Thebarda) [reply src/Listing/index.tsx]: yes. For example, when we display loading / empty table message

**Pull request merged**